src/options/pug-class-notation.ts

```diff
@@ -0,0 +1,16 @@
+import { ChoiceSupportOption } from 'prettier';
+import { CATEGORY_PUG } from '.';
+
+export const PUG_CLASS_NOTATION: ChoiceSupportOption = {
+	since: '1.13.0',
+	category: CATEGORY_PUG,
+	type: 'choice',
+	default: 'literal',
+	description: 'Define how classes should be formatted.',
+	choices: [
+		{ value: 'as-is', description: 'Disables class formatting.' },
+		{ value: 'literal', description: 'Forces all valid classes to be printed as literals.' }
```

Comment:
Also we could switch these choices
I think this affects the order when using the CLI `--help` (not that anyone uses it 😅 but hey, doing it directly the right way 😁)

Reply:
should I also switch on the readme?

Reply:
Yeah, everywhere
So it will ever be
1. literal
2. attribute
3. as-is